environment.ts

```diff
@@ -0,0 +1,8 @@
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
+    production: false,
+    facebookphotosToken: 'EAAE3R0omZCJUBAHvuLABjVTnHiHtVQrtktFRcNB8cjscTC3hZBsx2k9urAEcbMWJeF7HVb48oDaS99glfxxSRSjUpyZCtAWyGLs3ZAqTEyYs6PKVwVEcFRWY4Fi105kgURZAMRPEltg35adsvnQnhpdKYQ0yhaIqtN6ZALJDxNl9Bn9zZCXL6Kx7p3aKm6JlY8repc4U3QHCwZDZD',
+    facebookUrl: 'https://www.facebook.com/mengyang.li.775/photos?lst=100013809856212%3A100013809856212%3A1536804807&source_ref=pb_friends_tl'
```

Comment:
This looks cool, could you shall with us in the discussion what do lst and source_ref refer to?
Just a brief introduction will do.
For your reference, the string after '?' is called a query in an URL when it is sent to the backend, the backend will be able to retrieve the data by 'response.query', the return datatype will be a Object.
```javascript
{
  lst: "100013809856212%3A100013809856212%3A1536804807",
  source_ref: "pb_friends_tl"
}